src/Tasks/Microsoft.NET.Build.Tasks/IPackageResolver.cs

```diff
@@ -8,5 +8,6 @@ namespace Microsoft.NET.Build.Tasks
     public interface IPackageResolver
     {
         string GetPackageDirectory(string packageId, NuGetVersion version);
+        string GetPackageDirectory(string packageId, NuGetVersion version, out string packageRoot);
```

Comment:
> If someone tries, and they get broken in the future, they need to fix themselves.

My experience with these statements is that we make them while building v1 and then get blocked from actually breaking *anything* in v2 no matter how bizarre the use case. I'd rather make things explicitly internal to just avoid having to even discuss such a policy.


> The reason we can't make these types 'internal' and use IVTA right now is because #55 isn't fixed.

The current skip-signing condition is too aggressive. Public signing works fine on non-Windows and would allow us to use IVTA with the same key across all platforms.